CHANGELOG.md

```diff
@@ -1,3 +1,81 @@
+### citus v9.3.0 (May 6, 2020) ###
+
+* Adds `update_distributed_table_colocation` UDF to update colocation of tables
+
+* Adds propagation of  `ALTER ROLE .. SET` statements
```

Comment:
Two blank spaces in here. Here is the current line followed by the one with correct number oof spaces:

```
 * Adds propagation of  `ALTER ROLE .. SET` statements
 * Adds propagation of `ALTER ROLE .. SET` statements
```